tensorpipe/test/channel/channel_test.h

```diff
@@ -73,6 +70,14 @@ class DataWrapper<tensorpipe::CudaBuffer> {
     }
   }
 
+  explicit DataWrapper(DataWrapper&& other)
```

Comment:
Nit: doesn't the "rule of 5" say that whenever one defines a move constructor one should also define a move assignment operator?